ern-container-gen-ios/src/hull/ElectrodeContainer/ElectrodeReactNative.h

```diff
@@ -101,23 +105,29 @@ NS_ASSUME_NONNULL_BEGIN
                            properties:(NSDictionary * _Nullable)properties;
 
 /**
- Returns a react native miniapp (from a JSBundle) inside a view controller.
+ Returns a react native miniapp (from a JSBundle).
 
  @param name The name of the mini app, preferably the same name as the jsbundle
  without the extension.
  @param properties Any configuration to set up the mini app with.
- @return A UIViewController containing the view of the miniapp.
+ @return a UIView of the miniapp.
  */
 - (UIView *)miniAppViewWithName:(NSString *)name
                      properties:(NSDictionary *_Nullable)properties;
 
 /**
+ Returns a react native miniapp (from a JSBundle).
+
  @param name The name of the mini app, that is registered with the AppComponent.
  @param properties initialprops for a React Native miniapp.
  @param sizeFlexibilty defines size flexibility type of the root view
- @return UIView
+ @param delegate of the root view that handles intrinsic size updates
+ @return a UIView of the miniapp.
  */
-- (UIView *)miniAppViewWithName:(NSString *)name properties:(NSDictionary *_Nullable)properties sizeFlexibility:(NSInteger)sizeFlexibilty;
+- (UIView *)miniAppViewWithName:(NSString *)name
+                     properties:(NSDictionary *_Nullable)properties
+                sizeFlexibility:(NSInteger)sizeFlexibilty
+                       delegate:(id<MiniAppViewDelegate> _Nullable)delegate;
```

Comment:
could you write a new method for this? this might break the existing feature which is using this method

Reply:
done